Cakes-Co/index.html

```diff
@@ -7,10 +7,127 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="/Cakes-Co/style.css">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <section class="logobar">
+            <div class="logoimg">
+                <img class="logoimgclass" src="Cakes Images/images.png" alt="Our Logo">
+
+            </div>
+            <div class="logotxt">
+                <h3>The Best Cake in town delevered to your door, Just with our Cake Store.</h3>
+            </div>
+           
+        </section>
+        
+        <nav class="navclass">
+            <ul class="ulclass">
+                <li class="liclass">
+                    <a href="#">Home</a>
+                </li>
+                <li class="liclass">
+                    <a href="#">Cake</a>
+                </li>
+                <li class="liclass">
+                    <a href="#">OrderIn</a>
+                </li>
+                <li class="liclass">
+                    <a href="#">Lessons</a>
+                </li>
+                <li class="liclass">
+                    <a href="#">About</a>
+                </li>
+            </ul>
+
+        </nav>
+
+
+
+
+    </header>
+    <!--<section class="container"> -->
+      
+      <article>
+        
+        <section class="firstart">
```

Comment:
look at using css grid to make this mobile first

Reply:
Kindly Can you review my code changes, and provide me with your advice in your Feedback.